src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -14,6 +14,7 @@
 using System.Management.Automation.Internal;
 using System.Diagnostics.CodeAnalysis;
 using System.Security;
+using System.Security.Permissions;
```

Comment:
`SecurityZone` is in the namespace `System.Security` ... so this one is not needed.